src/components/input/index.js

```diff
@@ -1 +1 @@
-module.exports = require('./code/index');
+module.exports = require('./input.jsx');
```

Comment:
Some components still have an `index.js` and some don't, can we be consistent? (by the looks of things, if the expectation is that you include the built file out of `dist/`, they're not needed)